<launch>

	<node pkg="AutoPatrol" type="testAP" name="testAP" args="" output="screen" >
		<rosparam file="$(find AutoPatrol)/launch/test.yaml" command="load"/> 
  	</node>

	
</launch>


<!-- 用于暂存调整
launch-prefix="/usr/bin/xterm -e"
<node pkg="AutoPatrol" type="AutoPatrol_main" name="AP" args="" output="screen" >
  	</node>







-->
